lib/src/native/realm_core.dart

```diff
@@ -326,6 +326,47 @@ class _RealmCore {
     return RealmResultsHandle._(pointer);
   }
 
+  RealmResultsHandle queryTable(Realm realm, int classKey, String query, List<Object> args) {
```

Comment:
should we call it `queryClass` or `queryByClass`. Core calls it `table` but we don't work with tables but classes and are passing classkeys. Also we have `getClassMetadata` and not getTableMetadata so I suggest we go with `queryClass`. What do you think?

Reply:
Fixed